src/Nancy.Serialization.JsonNet/Helpers.cs

```diff
@@ -2,8 +2,39 @@
 {
     using System;
 
+    using Nancy.Json;
+
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+
     internal static class Helpers
     {
+        public static JsonSerializer DefaultSerializer()

```

Comment:
Is there really a need for this class? Why can't it say in the `JsonSerializer.CreateDefault` method?